woocommerce/woocommerce
# Create variation items block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #39455



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure that `New product editor` feature is `unchecked` under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features` and `product-variation-management` feature is `true` under `/wp-admin/tools.php?page=woocommerce-admin-test-helper` features tab
2. Create a variable product using the classic editor and generate some variations
3. Save the variable product id
4. Go to `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features` and `check` the `New product editor` feature
5. Go to `/wp-admin/admin.php?page=wc-admin&path=/product/{productId}&tab=variations` and replace the `{productId}` by the id you saved in point 3
6. Under the `Variations` section, the product variations generated in the point 2 should be shown



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
